tests/web-ui-smoke.sh

```
@@ -0,0 +1,89 @@
+LOG=$(mktemp)
+trap "rm -f $LOG" EXIT
+
+cd "$(dirname "$0")/.." || exit 1
+
+# Boot the app in the background on the chosen port.
+python3 app.py --port "$PORT" > "$LOG" 2>&1 &
+APP_PID=$!
+trap "kill $APP_PID 2>/dev/null; rm -f $LOG" EXIT
+
+# Wait up to 15s for /api/workspaces to respond.
+for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)